rampwf/utils/cli/testing.py

```diff
@@ -92,7 +93,9 @@ def main(submission, ramp_kit_dir, ramp_data_dir, data_label,
         ]
     else:
         submission = [submission]
-
+    if not os.path.exists("utils"):
+        os.makedirs("utils")
+    sys.path.append(os.path.join(ramp_kit_dir, "utils"))
```

Comment:
Wouldn't it be better to insert the path at the beginning of `sys.path`? This would mimick the fact that python first searches local modules before searching for libraries, although I am not completely sure.

Reply:
I am not sure either, I think that is recommended to do .append()